### DISCUSSION TITLE
Cryptomator and login keychain

### DISCUSSION THREAD

**Discussion by maggiorante** [iterate-ch/cyberduck#16092] [Q&A] [answered]
Hi, I just started using Cryptomator within Cyberduck. I disabled the option "Use Keychain" (it was already disabled actually) but every time i create a vault from inside the explorer the passphrase is added to my keychain. Have I misunderstood what this option does? I thought it meant I had to always be prompted to insert the password to unlock my vaults.

Thanks for any clarification!

**Comment by tatze96** [comment, marked as answer]
The _Use Keychain_ option deactivates the corresponding checkmark in the vault's passphrase prompt. Is it possible that you accidentally activated the checkbox within the passphrase prompt?

If you want Cyberduck to prompt you for the password each time you access the respective vault again, remove the corresponding entries from _Keychain.app_ (macOS) or _Windows Credential Manager_.

**Reply by maggiorante** [reply]
I did not look into this any further but I could fix this by removing the entries from wincred. I could have done something wrong the first time but idk.

Thanks for the help!
